app/components/Thread/index.jsx

```diff
@@ -11,6 +11,8 @@ const mui = require('material-ui');
 const md = require('../SmartEditor/markdown');
 const LoginStore = require('../../stores/LoginStore');
 
+require('./style.less');
```

Comment:
In that case I suggest you'd better create a new issue, describe the symptom, and make a new branch to solve it.